src/app/(private)/data-sources/[id]/DataSourceDashboard.tsx

```diff
@@ -138,42 +137,41 @@ export default function DataSourceDashboard({
               <BreadcrumbItem>{dataSource.name}</BreadcrumbItem>
             </BreadcrumbList>
           </Breadcrumb>
-          <h1 className="text-4xl font-medium tracking-tight">
+          <h1 className="text-3xl font-bold tracking-[-0.0085em]">
```

Comment:
Hey, I noticed the headings on this page have a different style (font weight and letter spacing) than headings across the app.

I implemented them the way they're designed but if it's better to keep them consistent with other headings, let me know (or the other way around - if you wanna change the styling of the headings in the whole app).

@Arbyhisenaj